subsys/mpsl/Kconfig

```
@@ -25,6 +25,15 @@ config MPSL_TIMESLOT_SESSION_COUNT
 	help
 	  Maximum number of timeslot sessions.
 
+config MPSL_ASSERT_HANDLER
+	bool "Application Defined Assertion Handler"
+	help
+	  This option enables an application-defined sink for the
+	  MPSL assertion mechanism. This must be defined in
+	  application code as void \"bmpsl_assert_handle(char \*, int)\"
+	  and will be invoked whenever the MPSL code encounters
+	  an unrecoverable error.
```

`\"bmpsl_assert_handle(char \*, int)\"` looks a bit strange. Maybe make it simpler?
```suggestion
	  This option enables an application-defined sink for the
	  MPSL assertion mechanism. This must be defined in
	  application code and will be invoked whenever the 
	  MPSL code encounters an unrecoverable error.
```